tests/controller_tests/test_filters.py

```diff
@@ -197,16 +197,23 @@ def test_get_outcome_alphabetical_sorting(seeded_test_client):
     assert response.json == ["Culture", "Economy", "Place", "Transport"]
 
 
-def test_get_regions(seeded_test_client):
+def test_get_geospatial_regions(seeded_test_client):
     """Asserts successful retrieval of regions."""
 
     response = seeded_test_client.get("/regions")
 
     assert response.status_code == 200
     response_json = response.json
 
-    assert response_json
-    assert all(isinstance(region, str) for region in response_json)
+    assert all("name" in region for region in response_json)
+    assert all(isinstance(region["name"], str) for region in response_json)
+
+    assert all("id" in region for region in response_json)
+    assert all(isinstance(region["id"], str) for region in response_json)
```

Comment:
Somewhat redundant given we're doing a full assert on these things below 😅  but all good